lldb/tools/lldb-dap/lldb-vscode/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "lldb-vscode",
+	"name": "lldb-dap",
```

Comment:
This and others are still in an "lldb-vscode" folder, that doesn't seem right.

Reply:
This and disassembly.json are still in the wrong place. I think move everything up one level and delete the lldb-vscode folder. As the instructions refer to package.json being in the same directory as the readme.

Reply:
Thanks, I had missed this comment.